swiftpkg/internal/repo_rules.bzl

```diff
@@ -70,8 +70,8 @@ def _gen_build_files(repository_ctx, pkg_ctx):
     for target in pkg_info.targets:
         # Unfortunately, Package.resolved does not contain test-only external
         # dependencies. So, we need to skip generating test targets. If a target
-        # does not have any product memberships, it is a testonly
-        if target.type == "test" or len(target.product_memberships) == 0:
+        # does not have any product memberships, it is a testonly.
```

Comment:
Since we no longer check `target.product_memberships`, does this comment still hold?